modules/browser/BrowserCapabilities.js

```diff
@@ -155,16 +156,16 @@ export default class BrowserCapabilities extends BrowserDetection {
      * 50ms to execute on the main thread.
      */
     supportsPerformanceObserver() {
-        return typeof window.PerformanceObserver !== 'undefined'
-            && PerformanceObserver.supportedEntryTypes.indexOf('longtask') > -1;
+        return window.PerformanceObserver
+            && window.PerformanceObserver.supportedEntryTypes.indexOf('longtask') > -1;
     }
 
     /**
      * Checks if the current browser supports audio level stats on the receivers.
      */
     supportsReceiverStats() {
-        return typeof window.RTCRtpReceiver !== 'undefined'
-            && Object.keys(RTCRtpReceiver.prototype).indexOf('getSynchronizationSources') > -1;
+        return window.RTCRtpReceiver
+            && 'getSynchroniziationSourceS' in window.RTCRtpReceiver.prototype;
```

Comment:
```suggestion
            && 'getSynchroniziationSources' in window.RTCRtpReceiver.prototype;
```

Reply:
so... why didn't this blow up?

Reply:
I don't think we have coverage for that alas.